snapcraft/plugins/v1/_ros/rosdep.py

```diff
@@ -218,6 +218,8 @@ def _run(self, arguments):
             self._rosdep_install_path, "usr", "lib", "python2.7", "dist-packages"
         )
 
+        env["ROS_PYTHON_VERSION"] = "2"
```

Comment:
Given that this class is used by both catkin and colcon plugins, it makes me a little nervous to hard-code this here. Are we certain it won't have bad ramifications for colcon, which is python 3?

Reply:
This var is only set for the env in which `rosdep` is run. Note that rosdep sets it to `2` by default. This does not leak to a colcon env and afaik colcon does not uses this env, so I believe we're good. Furthermore `PYTHONPATH` is hardcoded to a Python 2 path literally 3 lines above.

Reply:
Yeah fair point, okay.